specification/loadtestservice/resource-manager/Microsoft.LoadTestService.Draft/stable/2022-12-01/loadtestservice.json

```diff
@@ -0,0 +1,1021 @@
+      "type": "object",
+      "description": "The response of a OutboundEnvironmentEndpoint list operation.",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "The OutboundEnvironmentEndpoint items on this page",
+          "items": {
+            "$ref": "#/definitions/OutboundEnvironmentEndpoint"
+          }
+        },
+        "nextLink": {
+          "type": "string",
+          "format": "uri",
+          "description": "The link to the next page of items"
+        }
+      },
+      "required": [
+        "value"
+      ]
+    },
+    "QuotaBucketRequest": {
+      "type": "object",
+      "description": "Request object of new quota for a quota bucket.",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/QuotaBucketRequestProperties",
+          "description": "New quota request request properties.",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "QuotaBucketRequestProperties": {
+      "type": "object",
+      "description": "New quota request request properties.",
+      "properties": {
+        "currentUsage": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Current quota usage of the quota bucket.",
+          "minimum": 0
+        },
+        "currentQuota": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Current quota limit of the quota bucket.",
+          "minimum": 0
+        },
+        "newQuota": {
+          "type": "integer",
+          "format": "int32",
+          "description": "New quota limit of the quota bucket.",
+          "minimum": 0
+        },
+        "dimensions": {
+          "$ref": "#/definitions/QuotaDimensions",
+          "description": "Dimensions for new quota request."
+        }
+      }
+    },
+    "QuotaDimensions": {
+      "type": "object",
+      "description": "Dimensions for new quota request.",
+      "properties": {
+        "subscriptionId": {
+          "type": "string",
+          "description": "Subscription Id dimension for new quota request of the quota bucket.",
+          "minLength": 1
+        },
+        "location": {
+          "type": "string",
+          "description": "Location dimension for new quota request of the quota bucket.",
+          "minLength": 1
+        }
+      }
+    },
+    "QuotaResource": {
+      "type": "object",
+      "description": "Quota bucket details object.",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/QuotaResourceProperties",
+          "description": "The resource-specific properties for this resource.",
+          "x-ms-client-flatten": true,
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ProxyResource"
```

Comment:
The definition of ProxyResource model is as following

```
"ProxyResource": {
      "title": "Proxy Resource",
      "description": "The resource model definition for a Azure Resource Manager proxy resource. It will not have tags and a location",
      "type": "object",
      "allOf": [
        {
          "$ref": "#/definitions/Resource"
        }
      ]
    },
```